lib/handler/configurator/proxy.c

```diff
@@ -103,6 +103,15 @@ static int on_config_websocket_timeout(h2o_configurator_command_t *cmd, h2o_conf
     return h2o_configurator_scanf(cmd, node, "%" SCNu64, &self->vars->conf.websocket.timeout);
 }
 
+static int on_config_websocket_http2(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, yoml_t *node)
+{
+    ssize_t ret = h2o_configurator_get_one_of(cmd, node, "OFF,ON");
+    if (ret == -1)
+        return -1;
+    ctx->globalconf->proxy.websocket_http2 = (int)ret;
+    return 0;
+}
```

Comment:
Would it make sense to change this to a configuration variable that changes the number of websocket connections being permitted for each HTTP/2 connection?

See also my comment above about states.